lib/sentry/context.ex

```diff
@@ -68,8 +192,52 @@ defmodule Sentry.Context do
     Process.get(@process_dictionary_key) || %{}
   end
 
+  @doc """
+  Adds a new breadcrumb to the `:breadcrumb` context, specific to the current
+  process.
+
+  Breadcrumbs are used to record a series of events that led to a specific
+  instance of an error. Breadcrumbs can contain arbitrary key data to assist in
+  understanding what happened before an error occurred.
+
+  ## Example
+
+      iex> Sentry.Context.add_breadcrumb(message: "first_event")
+      nil
+      iex> Sentry.Context.add_breadcrumb(%{message: "second_event", type: "auth"})
+      %{breadcrumbs: [%{:message => "first_event", "timestamp" => 1562007480}]}
+      iex> Sentry.Context.add_breadcrumb(%{message: "response"})
+      %{
+          breadcrumbs: [
+                %{:message => "second_event", :type => "auth", "timestamp" => 1562007505},
+                %{:message => "first_event", "timestamp" => 1562007480}
+              ]
+      }
+      iex> Sentry.Context.get_all()
+      %{
+          breadcrumbs: [
+                %{:message => "first_event", "timestamp" => 1562007480},
+                %{:message => "second_event", :type => "auth", "timestamp" => 1562007505},
+                %{:message => "response", "timestamp" => 1562007517}
+              ],
+          extra: %{},
+          request: %{},
+          tags: %{},
+          user: %{}
+      }
+  """
+  @spec add_breadcrumb(keyword() | map()) :: nil | map()
   def add_breadcrumb(list) when is_list(list) do
-    add_breadcrumb(Enum.into(list, %{}))
+    if Keyword.keyword?(list) do
+      list
+      |> Enum.into(%{})
+      |> add_breadcrumb
```

Comment:
Made this small change from `add_breadcrumb(Enum.into(list, %{}))` as this is a bit more idiomatic Elixir, but does exactly the same thing.